hibernate-core/src/main/java/org/hibernate/dialect/Oracle10gDialect.java

```diff
@@ -22,26 +22,50 @@
  * Boston, MA  02110-1301  USA
  */
 package org.hibernate.dialect;
+import java.sql.Types;
+
+import org.hibernate.cfg.Environment;
+import org.hibernate.dialect.function.StandardSQLFunction;
 import org.hibernate.sql.ANSIJoinFragment;
 import org.hibernate.sql.JoinFragment;
+import org.hibernate.type.StandardBasicTypes;
 
 
 /**
  * A dialect specifically for use with Oracle 10g.
  * <p/>
  * The main difference between this dialect and {@link Oracle9iDialect}
  * is the use of "ANSI join syntax".  This dialect also retires the use
- * of the <tt>oracle.jdbc.driver</tt> package in favor of 
+ * of the <tt>oracle.jdbc.driver</tt> package in favor of
  * <tt>oracle.jdbc</tt>.
  *
  * @author Steve Ebersole
  */
 public class Oracle10gDialect extends Oracle9iDialect {
 
 	public Oracle10gDialect() {
-		super();
+	    super();
```

Comment:
Why did you change the Oracle Dialect when this issue description is about Derby?